### PR TITLE
Fix grid

### PR DESCRIPTION
**Pull request by dansterenson**

### CODE REVIEW

File: client/src/components/mainContent.vue [hunk 1]
```diff
@@ -22,91 +22,81 @@
...
-        <v-col>
-          <v-card class="pa-2" outlined tile>
-            <p>Percent Change: {{ percentChange }}</p>
-          </v-card>
-        </v-col>
-      </v-row>
+        </v-flex>
+      </v-layout>
+
+<!--      <v-row class="mb-6" justify="center" no-gutters>-->
```
Comment (BenSterenson) [inline client/src/components/mainContent.vue:81]: why do u keep it?

Comment (dansterenson) [reply client/src/components/mainContent.vue:81]: Removed it

**Review by BenSterenson — APPROVED** [review]

File: client/src/components/mainContent.vue [hunk 2]
```diff
@@ -22,91 +22,61 @@
...
             <section class="charts">
               <bar-chart :names="commissionsNames" :values="commissionsValues" chart-name="Commissions"></bar-chart>
             </section>
           </v-card>
-        </v-col>
+        </v-flex>
 
-        <v-col>
-          <v-card class="pa-2" outlined tile>
+        <v-flex xd6 md3>
```
Comment (MPalarya) [inline client/src/components/mainContent.vue:64]: `xs`?

File: client/src/components/mainContent.vue [hunk 3]
```diff
@@ -22,91 +22,61 @@
...
             <section class="charts">
               <pie-chart :data-array="assetsArray" chart-name="Assets"></pie-chart>
             </section>
           </v-card>
-        </v-col>
+        </v-flex>
 
-        <v-col>
-          <v-card class="pa-2" outlined tile>
+        <v-flex xd6 md3 align="center">
```
Comment (MPalarya) [inline client/src/components/mainContent.vue:72]: `xs`?

**Pull request merged**